### PR TITLE
ci: release action

### PR DESCRIPTION
**Pull request by TorstenDittmann**
## What does this PR do?

- only build console on host build architecture
- build release using CI

## Test Plan

- manual

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Review by stnguyen90 — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    # This is a separate action that sets up buildx runner
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
```
Comment (christyjacob4) [inline .github/workflows/release.yml:30]: Why do we have it twice though?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@v4
+      with:
+        images: appwrite/appwrite
+        tags: |
+          type=semver,pattern={{version}}
+          type=semver,pattern={{major}}
+          type=semver,pattern={{major}}.{{minor}}
+
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply .github/workflows/release.yml:47]: Can we be more explicit here for readability

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
Looks good. Please address the comments

File: .github/workflows/release.yml (hunk 1 above)
Comment (TorstenDittmann) [reply .github/workflows/release.yml:30]: good question 😄

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**